fix inconsistent linting results

## Type of Change
- [ ] New Feature Request
- [ ] Improvement / Suggestion
- [ ] Bug
- [x] Other (please clarify below)

## Summary
Noticing that the linter running in Jenkins is giving different results than locally.

CircleCI
https://circleci.com/gh/ProvidenceGeeks/website-frontend/387
```shell
ERROR in ./components/social-follow-links/social-follow-links.jsx

/root/repo/src/components/social-follow-links/social-follow-links.jsx
  17:1  error  Line 17 exceeds the maximum line length of 200  max-len

✖ 1 problem (1 error, 0 warnings)
```

Locally
```shell
$ yarn lint
yarn run v1.12.3
$ eslint *.js ./src/**/**/*.spec.*
✨  Done in 1.47s.
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -1,10 +1,12 @@
 {
+  "parser": "babel-eslint",
   "parserOptions": {
+    "allowImportExportEverywhere": true,
+    "sourceType": "module",
     "ecmaVersion": 6,
     "ecmaFeatures": {
       "jsx": true
-    },
-    "sourceType": "module"
+    }
   },
   "extends": [
     "plugin:react/recommended"
@@ -142,7 +144,7 @@
     "max-nested-callbacks": [2, {"maximum":8}],
     "new-cap": 2,
     "new-parens": 2,
-    "newline-after-var": 2,
+    "newline-after-var": 0,
     "no-array-constructor": 2,
     "no-continue": 0,
     "no-inline-comments": 0,

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "clean": "rimraf ./build && rimraf ./reports",
     "build": "npm run clean && npm run lint && webpack --config ./webpack.config.prod.js",
     "develop": "webpack-dev-server --open --config ./webpack.config.develop.js",
-    "lint": "eslint *.js ./src/**/**/*.spec.*",
+    "lint": "eslint \"*.js\" \"./src/**/**/*.js*\"",
     "release": "npm run clean && npm run lint && webpack --config ./webpack.config.release.js",
     "serve": "npm run build && http-server ./build -o --proxy http://stage.pvdgeeks.org",
     "test": "npm run clean && jest --config ./jest.config.json"
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "babel-core": "^6.26.0",
+    "babel-eslint": "^10.0.1",
     "babel-jest": "^21.2.0",
     "babel-loader": "^7.1.2",
     "babel-preset-env": "^1.6.0",

